### PR TITLE
Added description for `github.secret_source`

### LINKED ISSUE #17752: Document `github.secret_source` context value
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

### What part(s) of the article would you like to see updated?

The property name `github.secret_source` is not documented. As far as I can understand, the value can be `None | Actions | Dependabot`. It would be good to confirm that. It is included in the example, just not in the table that documents all properties. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #17752

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added description for parameter `github.secret_source`. 

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
@codewithdev Thanks for opening this PR addressing one of our `help wanted` issues. I'll get this triaged for review! ⚡

**Review by jhosman — COMMENTED** [review]
👋 @codewithdev - Thanks! I'm pushing some additional changes, and we're going to get another review from someone internally before we merge this. Thanks for submitting this PR! ✨

File: content/actions/learn-github-actions/contexts.md
Comment (jhosman) [inline content/actions/learn-github-actions/contexts.md:214]:
```suggestion
{%- ifversion fpt or ghec or ghes > 3.3 or ghae > 3.3 %}
```

**Review by lucascosti — APPROVED** [review]
LGTM! 🚀

**Pull request merged**
